packages/front-end/components/Experiment/NewExperimentForm.tsx

```diff
@@ -92,11 +92,13 @@ export function getNewExperimentDatasourceDefaults(
   return {
     datasource: initialDatasource.id,
     exposureQueryId:
-      getExposureQuery(
+      initialDatasource.settings.defaultQueryId ??
```

Comment:
The way you have this now it will always use the default query, even if there is an initialValue set.  Are you sure that is the behavior that we need?

Reply:
You're right, this does prioritize the default ID over the current ID for when a query is being edited. 

I'll fix that.

Reply:
I've made the fix, so as it stands now, we call `getExposureQuery`, and if we pass in an `initialValue`, it will use that. (This would be the case for when the form is used for editing instead of creating). 
If the `initialValue` is `null`, it will use the `defaultId` if set, and will finally fallback to the default fallback that was always there if the defaultId has not been set.

Reply:
I've reverted to the original, simpler span display method. :+1: